### PR TITLE
Add EBL data from Dominguez et al. 2011

### LINKED ISSUE #6: Missing EBL data
One of the few parts missing from the original script is the EBL data file and the possibility of loading it to apply absorption.

### PR DESCRIPTION
**Pull request by HealthyPear**
closes #6

### CODE REVIEW

**Comment by HealthyPear** [comment]
@jsitarek would you like to review this? It comes from your old scripts linked [here](https://magic.mpp.mpg.de/public/magicop/) under "MAGIC PROPOSAL TOOLS / Source Detectability"

File: src/iact_estimator/io.py [hunk 1]
```diff
@@ -56,6 +56,12 @@ def load_ebl(ebl_file_path):
         Energy values.
     taus : `np.array`
         TBD.
+
+    Notes
+    -----
+    This is a legacy function which works
+    only wuth the default data file packages with
```
Comment (jsitarek) [inline src/iact_estimator/io.py:63]: with

**Review by jsitarek — APPROVED** [review]
looks good, just one typo, but since you are either way using gammapy, why not taking the EBL models from it?

**Comment by HealthyPear** [comment]
> looks good, just one typo, but since you are either way using gammapy, why not taking the EBL models from it?

yes I am going to use gammapy in the end for most of the estimations, this PR is a bit legacy: I'd like to make sure that the package replicates the old scripts first before the big refactoring.

File: src/iact_estimator/io.py (hunk 1 above)
Comment (HealthyPear) [reply src/iact_estimator/io.py]: fixed

**Pull request merged**